pkg/registry/core/service/storage/rest.go

```diff
@@ -153,7 +178,13 @@ func (rs *REST) NewList() runtime.Object {
 }
 
 func (rs *REST) Get(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
-	return rs.services.Get(ctx, name, options)
```

Comment:
this whole hunk should be reverted

Reply:
yes - this a left over since we had defaulting code there.